tools/test_containers_compose

```diff
@@ -9,4 +9,13 @@ for i in webui worker; do
     sudo docker-compose up -d &&
     (docker-compose ps --services --filter status=stopped | grep "^[[:space:]]*$") || (docker-compose logs; sudo docker-compose ps; exit 1)
   ) || exit 1;
+
+  if [[ $i == "webui" ]]; then
+    (
+      docker-compose exec webui /usr/share/openqa/script/clone_job.pl --from https://openqa.opensuse.org --host http://nginx:9526 1738950 || echo "Error executing a job";
```

Comment:
This job might be cleaned up on o3 at some point so don't rely on that.

Reply:
yes, this is a bad point of doing this in this way... should be a better way to solve this

Reply:
However we are testing that API can connect with the scheduler to launch tests...

Reply:
I might be missing context but relying on any data on o3 seems is a bad idea unless you're validating the deployment of this instance. This should use a fixture or create a job on the fly.

Reply:
Yes I agree

Reply:
It feels like you try to achieve too much here. I think having just a manual test properly defined in the commit message is enough. Having an automatic test that relies on o3 is too much

Reply:
Now is solved with an easy way